common/src/main/resources/assets/bingo/lang/en_us.json

```diff
@@ -313,5 +316,23 @@
   "bingo.board_corner.upper_left": "Upper-left",
   "bingo.board_corner.upper_right": "Upper-right",
   "bingo.board_corner.lower_left": "Lower-left",
-  "bingo.board_corner.lower_right": "Lower-right"
+  "bingo.board_corner.lower_right": "Lower-right",
+
+  "bingo.ordinal.1": "1st",
+  "bingo.ordinal.2": "2nd",
+  "bingo.ordinal.3": "3rd",
+  "bingo.ordinal.4": "4th",
+  "bingo.ordinal.5": "5th",
+  "bingo.ordinal.6": "6th",
+  "bingo.ordinal.7": "7th",
+  "bingo.ordinal.8": "8th",
+  "bingo.ordinal.9": "9th",
+  "bingo.ordinal.10": "10th",
+  "bingo.ordinal.11": "11th",
+  "bingo.ordinal.12": "12th",
+  "bingo.ordinal.13": "13th",
+  "bingo.ordinal.14": "14th",
+  "bingo.ordinal.15": "15th",
+  "bingo.ordinal.16": "16th",
+  "bingo.ordinal.generic": "%sth"
```

Comment:
Instead of hardcoding a bunch of these, something like [this function](https://stackoverflow.com/a/6810409/8840278) should probably be used instead.

Reply:
That only works for English. I wanted this to be translatable.

Reply:
The current solution does not work for English however. For example, 21 and 22 should be 21st and 22nd respectively, not 21th and 22th.

Reply:
Can you make them not numerical then? So like "first", "second", "third", and so forth.